packages/@ama-sdk/swagger-builder/src/core/swagger-spec-merger.ts

```diff
@@ -264,7 +263,7 @@ export class SwaggerSpecMerger {
     const envelops = await Promise.all(this.specs.map((s) => s.getEnvelop()));
     return envelops
       .reduce<Record<string, unknown>>((acc, e) => {
-        const info = {...(acc.info as Record<string, unknown> || {}), ...(e.info || {})};
```

Comment:
I wonder if the linter got tricked into thinking that `acc.info` cannot be undefined because of the explicit cast
`acc.info` is clearly undefined in the first iteration, not sure if this is a problem though

Reply:
actually `const a = {...undefined}` is equivalent to `const a = {}` and does not raise error